cppwinrt/code_writers.h

```diff
@@ -474,25 +468,15 @@ namespace cppwinrt
             auto format = R"(    template <> inline constexpr guid guid_v<%>{ % }; // %
 )";
 
-            w.write(format,
-                type,
-                bind<write_guid_value>(guid),
-                bind<write_guid_comment>(guid));
+            w.write(format, type, bind<write_guid_value>(guid), bind<write_guid_comment>(guid));
         }
         else
         {
             auto format = R"(    template <%> inline constexpr guid guid_v<%>{ pinterface_guid<%>::value };
     template <%> inline constexpr guid generic_guid_v<%>{ % }; // %
 )";
 
-            w.write(format,
```

Comment:
I find these `.write(...)` calls to be harder to match with the format string when the args get packed, and easier when they are one-per-line. But why is this happening even though `BinPackArguments: false` is in the format file?

Reply:
The column length cap is relatively long so many of these fit onto a single line now.

Reply:
I let the default style shorten the column cap back to 120 which fixed _most_ of these.  I'll also check and see if there's a way to special-case by method name.

Reply:
To be fair, when it's a shorter list, like 2-3 args, the readability on one line isn't bad. It was just a bit rough when I was having to count more than 5-6 args and they were starting to cram onto separate lines.

Finally figured out that BinPackArguments = false still allows packing the args, as long as they fit on a single line. So that probably explains it.